src/test/java/io/vertx/httpproxy/impl/ParseUtilsTest.java

```diff
@@ -0,0 +1,55 @@
+package io.vertx.httpproxy.impl;
+
+import io.vertx.ext.unit.TestContext;
+import io.vertx.httpproxy.TestBase;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class ParseUtilsTest extends TestBase {
```

Comment:
```suggestion
public class ParseUtilsTest {
```

It seems there is no need to create an instance of Vertx before each test